app/sablier.go

```diff
@@ -37,6 +37,16 @@ func Start(ctx context.Context, conf config.Config) error {
 		return err
 	}
 
+	// Extract Docker ping function for health checks if we're using Docker
```

Comment:
We should instead add a new method to every provider to check the current provider health.

I have the following names in mind:
- `Probe`
- `Ping`
- `Health`

Then this function should only call the provider healthchecking method.